content/publications/_index.md

```diff
@@ -2,6 +2,8 @@
 bibliography:
 - title: 2020
   items:
+  - matsubara-levorato-2020-emdl
+  - matsubara-levorato-2020-icpr
```

Comment:
These two references are [still][prev] missing from Marco’s bibliography
in `content/people/levorato.md`.

[prev]: https://github.com/uci-iasl/website/pull/34#discussion_r471154513